<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="7" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="33050" started-at="2019-12-05T13:36:41 EST" finished-at="2019-12-05T13:37:14 EST">
    <groups>
    </groups>
    <test name="Test" duration-ms="33050" started-at="2019-12-05T13:36:41 EST" finished-at="2019-12-05T13:37:14 EST">
      <class name="testcases.Testcase_Transferfunds">
        <test-method status="PASS" signature="transferfunds()[pri:0, instance:testcases.Testcase_Transferfunds@6121c9d6]" name="transferfunds" duration-ms="7520" started-at="2019-12-05T13:36:41 EST" finished-at="2019-12-05T13:36:48 EST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- transferfunds -->
        <test-method status="PASS" signature="quitBrowser()[pri:0, instance:testcases.Testcase_Transferfunds@6121c9d6]" name="quitBrowser" is-config="true" duration-ms="679" started-at="2019-12-05T13:37:13 EST" finished-at="2019-12-05T13:37:14 EST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
      </class> <!-- testcases.Testcase_Transferfunds -->
      <class name="testcases.Testcase_AddPAyee">
        <test-method status="PASS" signature="validateAddPayee()[pri:0, instance:testcases.Testcase_AddPAyee@87f383f]" name="validateAddPayee" duration-ms="7745" started-at="2019-12-05T13:36:48 EST" finished-at="2019-12-05T13:36:56 EST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAddPayee -->
      </class> <!-- testcases.Testcase_AddPAyee -->
      <class name="testcases.Testcase_Login">
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String)[pri:0, instance:testcases.Testcase_Login@4eb7f003]" name="login" duration-ms="5181" started-at="2019-12-05T13:36:57 EST" data-provider="credential" finished-at="2019-12-05T13:37:02 EST">
          <params>
            <param index="0">
              <value>
                <![CDATA[username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String)[pri:0, instance:testcases.Testcase_Login@4eb7f003]" name="login" duration-ms="5359" started-at="2019-12-05T13:37:02 EST" data-provider="credential" finished-at="2019-12-05T13:37:08 EST">
          <params>
            <param index="0">
              <value>
                <![CDATA[username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String)[pri:0, instance:testcases.Testcase_Login@4eb7f003]" name="login" duration-ms="5288" started-at="2019-12-05T13:37:08 EST" data-provider="credential" finished-at="2019-12-05T13:37:13 EST">
          <params>
            <param index="0">
              <value>
                <![CDATA[richa]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[abc]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Zero - Log in] but found [Zero - Account Summary]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Zero - Log in] but found [Zero - Account Summary]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at testcases.Testcase_Login.login(Testcase_Login.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- testcases.Testcase_Login -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
